webapp/src/containers/genePage/transcriptInlineViewer.js

```diff
@@ -46,8 +46,10 @@ class TranscriptViewer extends Component {
     return (
       <div id="genomeViewer">
         {/*<iframe id="genomeFrame" className={style.jbrowse} src={externalJbrowseUrl}/>*/}
-        <a href={externalJbrowseUrl.replace('overview.html','index.html')}>Genome Viewer<i className="fa fa-link" /> </a>
-        <a href={externalJbrowseUrl}>Overview<i className="fa fa-link" /> </a>
+        <a href={externalJbrowseUrl.replace('overview.html','index.html')} className="btn btn-warning btn-sm">Browse <i className="fa fa-search" /></a>
+        &nbsp;
+        &nbsp;
```

Comment:
Are these just to add space between the buttons? Can that be done with CSS instead if needed?